content/community/news/science-updates/2023-03-10-NIH-Renewal.md

```diff
@@ -0,0 +1,23 @@
+---
+date: "2023-03-10T00:00:00+00:00"
+title: 'Open Grant writing'
+tags: ["force field", "open science", "Open Force Field", "OpenFF" ]
+categories: ["science"]
+draft: true
+description: 'NIH Renewal'
+slug: 'NIH-renewal-2023-03-10'
+weight: 10
+markup: markdown
+thumb: "OpenFF-Initiative.png"
+author: "Michael Shirts"
+---
+
+We've been very thankful to recieve significant funding from the NIH Institute of General Medical Sciences for OpenFF over the last three years 
+from the Focused Technology Research and Development program. We recently submitted a proposal to renew our funding for four more years, once the 
+current grant ends next year. In the interest, of open science, we are [posting the scientific parts of the proposal](../../../science/research/2023_03_06_Open_Force_Field_Initiative_NIH_R01.pdf).
+
+We feel that overall, as part of an open science project, it's better to be open about our plans rather than keep them secret. 
```

Comment:
We don't need to explain any of this. We just need to lead by example.

I'd cut this whole paragraph.